messages/list/edge_applications/messages.go

```diff
@@ -0,0 +1,8 @@
+package edge_applications
+
+var (
+	ListUsage            = "edge_applications [flags]"
+	ListShortDescription = "Displays edge applications in a list"
+	ListLongDescription  = "Displays all your edge applications in a list"
+	ListHelpFlag         = "Displays more information about the list subcommand"
```

Comment:
"Displays more information about the list command"

it is no longer a subcommand